### PR TITLE
Added bitsong chain support

### PR DESCRIPTION
**Pull request by betterclever**

### CODE REVIEW

**Comment by tombeynon** [comment]
Looks great @betterclever! Could you just add a deploy.yml for Akash, and a README entry similar to this PR https://github.com/ovrclk/cosmos-omnibus/pull/47

The deploy.yml can be the same as any other chain, just change the image and env variables.

Can you just clarify if the validate genesis step fails for Bitsong? I see you have `VALIDATE_GENESIS=0` so just want to make sure that's intentional.

**Comment by betterclever** [comment]
Yes. The validate genesis is indeed failing, due to wrong order of the `ubtsg` and `btsg` currencies but I verified that it runs just fine.
Also, I asked a question on Discord but cross-posting here for visibility.

I assume the Docker image on the ovrclk ghcr, is only pushed after CI is run after publishing from the master branch. How do I split my PR then? Or do I optimistically make it assuming the CI image will be made just fine 🤔
I wished to do a test before, can we not make some sort of PR CI to do this validation. Happy to work on it.

File: bitsong/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+version: "2.0"
+
+services:
+  node:
+    image: ghcr.io/ovrclk/cosmos-omnibus:v0.0.10-bitsong-v0.8.0
+    env:
+      - MONIKER=node_1
+      - CHAIN_JSON=https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/chain.json
+      - CHAIN_ID=bitsong
```
Comment (tombeynon) [inline bitsong/deploy.yaml:10]: I think this CHAIN_ID is incorrect, doesn't match the chain.json. Should just be able to remove this line

**Review by tombeynon — CHANGES_REQUESTED** [review]

File: bitsong/deploy.yaml (hunk 1 above)
Comment (betterclever) [reply bitsong/deploy.yaml]: Done

**Review by tombeynon — APPROVED** [review]
🚀

**Pull request merged**